packages/drivers/odsp-driver/src/opsCaching.ts

```diff
@@ -0,0 +1,167 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// ISequencedDocumentMessage
+export interface IMessage {
+    sequenceNumber: number;
+}
+
+type CacheEntry = (IMessage | undefined)[];
+
+interface IBatch {
+    currentBatchSize: number;
```

Comment:
As this is part of batch itself, I think we can just name it as size or batchsize. I would prefer removing "current".

Reply:
I'll reverse it and call it 'remainingSlots' as that's better represents the intention here.